packages/api/lib/writeRecords/write-granules.js

```diff
@@ -453,28 +459,33 @@ const _writeGranuleFiles = async ({
 const _writeGranuleRecords = async (params) => {
   const {
     postgresGranuleRecord,
-    apiGranuleRecord,
     knex,
     esClient = await Search.es(),
     granuleModel,
     executionCumulusId,
     granulePgModel,
     writeConstraints = true,
   } = params;
+  let { apiGranuleRecord } = params;
   let pgGranule;
+  let pgGranuleResult;
 
   log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
   log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
 
   try {
     await createRejectableTransaction(knex, async (trx) => {
-      pgGranule = await _writePostgresGranuleViaTransaction({
+      pgGranuleResult = await _writePostgresGranuleViaTransaction({
```

Comment:
If we're only using `pgGranuleResult` in this scope I think we can make it a `const` here instead of a variable in the upper scope.

Reply:
Ah I see. That make sense. I don't love having `pgGranuleResult` and `pgGranule` but it's pretty nit-picky. I think a couple options would be 
1. Renaming `pgGranuleResult` something like `writeGranuleResult` to make it clear it's not a PG Granule record
2. Seeing if we can eliminate one or the other in the outer scope. It'd be nice to just have one for clarity

Reply:
I am ok with changing it `writePgGranuleResult`. Also, it's possible to not have `pgGranule` at all, and just use `writePgGranuleResult.pgGranule` in all the places, but that seems a bit long.

Reply:
I've made this change to `writePgGranuleResult`, let me know if you also want me to get rid of `pgGranule` and use `writePgGranuleResult.pgGranule` everywhere